resources/assets/js/designer/components/modals/modal-create-report-add.vue

```diff
@@ -0,0 +1,330 @@
+                <div class="drag-to-table" v-for="form in list">
+                    <i class="fas fa-ellipsis-v fa-sm"></i><i class="fas fa-ellipsis-v fa-sm"></i>
+                  {{form.form_field_full}}
+                </div>
+            </draggable>
+
+            <i class="seperator fas fa-arrow-right"></i>
+
+            <div class="table-wrapper">
+              <div class="d-flex head-wrapper">
+                <div class="table-head form-field">form FIELD</div>
+                <div class="table-head name-field">NAME</div>
+                <div class="table-head label-field">LABEL</div>
+                <div class="table-head type-field">TYPE</div>
+                <div class="table-head size-field">size</div>
+                <div class="table-head increment-field">AUTO INCREMENT</div>
+                <div class="table-head index-field">index</div>
+              </div>
+              <draggable class="table-target" v-model="list2" :options="{group:'forms'}">
+                <div class="d-flex row-wrapper" v-for="form in list2">
+                  <div class="form-field">
+                      <i class="fas fa-ellipsis-v fa-sm"></i><i class="fas fa-ellipsis-v fa-sm"></i>
+                    {{form.form_field}}
+                  </div>
+                  <div class="name-field">
+                    {{form.name}}
+                  </div>
+                  <div class="label-field">
+                    {{form.label}}
+                  </div>
+                  <div class="type-field">
+                    {{form.type}}
+                  </div>
+                  <div class="size-field">
+                    {{form.size}}
+                  </div>
+                  <div class="increment-field">
+                    {{form.auto_increment}}
+                  </div>
+                  <div class="index-field">
+                    {{form.index}}
+                  </div>
+                </div>
+              </draggable>
+            </div>
+          </div>
+        </div>
+    </form>
+
+    <template slot="modal-footer">
+      <b-button @click="onCancel" class="btn-outline-secondary btn-md">
+        CANCEL
+      </b-button>
+      <b-button class="btn-secondary text-light btn-md">
+        SAVE
+      </b-button>
+    </template>
+
+  </b-modal>
+</template>
+
+<script>
+import draggable from 'vuedraggable'
+
+export default {
+  components:{
+    draggable
+  },
+  data() {
+    return {
+      // form models here
+      'description':"Description",
+      'reportTable': "Report Table",
+      'formFields': "Form Fields",
+      'tableName': 'Table Name',
+      'type': 'Type',
+      'typeSelect': "View",
+      'typeSelectOptions':[
+        'View', 'Block'
+      ],
+      'dbConnection':"DB Connection",
+      'dbConnectionSelect': 'View',
+      'dbConnectionSelectOptions':[
+        'View', 'Block'
+      ],
+      list: [
+        {
```

Comment:
Please remove this dummy data. Do we know where the real data will come from? Is this something we can pull from the 3.0 tools?

Reply:
many of them are blank unless something has been assigned. I can just leave it blank for now.